testing/tests/DevExpress.viz.core.series/label.tests.js

```diff
@@ -1184,19 +1184,35 @@ QUnit.module('Layouted label', $.extend({}, environment, {
     }
 }));
 
-QUnit.test('simple shift', function(assert) {
+QUnit.test('Groups are added to the corresponding elements', function(assert) {
     const label = this.createLabel();
-    label.show();
 
+    label.show();
     label.shift(10, 10);
 
     const innerGroup = label._insideGroup;
+
     assert.ok(innerGroup);
-    assert.deepEqual(innerGroup._stored_settings, { translateX: 10 - 1, translateY: 10 - 2 });
     assert.equal(label._insideGroup.stub('append').lastCall.args[0], label._group);
     assert.equal(label._group.stub('append').lastCall.args[0], this.group);
 });
 
+QUnit.test('Label should be moved', function(assert) {
```

Comment:
or it's specified in a module name?
```suggestion
QUnit.test('shift method call should move a label on specified delta', function(assert) {
```

Reply:
Was fixed here: [49a79514db22d1cf7c7e60d512cae2bb1320f314].